db/schema.rb

```diff
@@ -38,4 +46,24 @@
     t.index ["room_id"], name: "index_shifts_on_room_id"
   end
 
+  create_table "users", force: :cascade do |t|
```

Comment:
Why are you creating a User model in a PR that talks about offspring model?
Why there is no belong_to  has_many in the model if the user is here?